workspace/src/main/kotlin/com/cosmotech/workspace/service/WorkspaceServiceImpl.kt

```diff
@@ -106,56 +109,72 @@ internal class WorkspaceServiceImpl(
     return result
   }
 
-  override fun findWorkspaceById(organizationId: String, workspaceId: String): Workspace {
+  override fun getWorkspace(organizationId: String, workspaceId: String): Workspace {
     return updateSecurityVisibility(getVerifiedWorkspace(organizationId, workspaceId))
   }
 
-  override fun createWorkspace(organizationId: String, workspace: Workspace): Workspace {
+  override fun createWorkspace(organizationId: String, workspaceCreateRequest: WorkspaceCreateRequest): Workspace {
```

Comment:
This method is broken. 
E.g: If I create a workspace and I specified a description in WorkspaceCreateRequest, its not saved.

Could you please check that and add tests on CRUD Workspace methods please ?